python/ray/dashboard/agent.py

```diff
@@ -111,11 +111,14 @@ def _init_non_minimal(self):
                 ),
             )  # noqa
         )
-        grpc_ip = "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0"
         try:
-            self.grpc_port = add_port_to_grpc_server(
-                self.server, build_address(grpc_ip, self.dashboard_agent_port)
+            add_port_to_grpc_server(
+                self.server, build_address(self.ip, self.dashboard_agent_port)
```

Comment:
Do you mind renaming `self.dashboad_agent_port` to `self.grpc_port` which is more clear.